includes/class-wc-payments-captured-event-note.php

```diff
@@ -66,7 +66,7 @@ public function generate_html_note(): string {
 
 		$html = '';
 		foreach ( $lines as $line ) {
-			$html .= self::HTML_BR . $line . PHP_EOL;
+			$html .= '<p>' . $line . '</p>' . PHP_EOL;
 		}
 
 		return '<div class="captured-event-details" style="line-height: 0.8;padding-top: 15px;">' . PHP_EOL
```

Comment:
Non blocking: I think we can remove the inline styles. The line-height has no effect and also we can take advantage of the native padding of the `<p>`. This is how it looks for me when I disable the inline styles with dev tools:

<img width="267" alt="Screenshot 2025-01-16 at 17 54 42" src="https://github.com/user-attachments/assets/44c35daf-1c1c-498b-96c8-e1feb0d81e68" />